jax/experimental/sparse_ops.py

```diff
@@ -403,6 +420,23 @@ def _coo_matvec_gpu_translation_rule(c, data, row, col, v, *, shape, transpose):
   xla.backend_specific_translations['gpu'][
       coo_matvec_p] = _coo_matvec_gpu_translation_rule
 
+def _coo_matvec_jvp_rule(primals_in, tangents_in, **params):
+  vals, rows, cols, vec = primals_in
+  sparse_mat_dot, _, _, vec_dot = tangents_in
```

Comment:
As above, I think we should assert that the underscored values are `ad.Zero`

Reply:
Done. I also added the assertions to `coo_matmat`.